Gemfile

```diff
@@ -34,14 +34,21 @@ else
   end
 
   case ENV['RAILS_VERSION']
+  when '6.0.0' # This will only be necessary until release 6.0.1 is published
+    gem 'sass-rails', '~> 5'
+    gem 'webpacker' # This is necessary or the generator will break
   when /^5\./
     gem 'capybara', '~> 2.18.0'
-  when /^4\.2/
+    gem 'sass-rails', '~> 5'
+  when /^4\./
```

Comment:
This needs to be removed, Core Components are no longer required to support Rails 4.y.z releases.